purchasing/templates/layout.html

```diff
@@ -60,6 +60,9 @@
   {% assets "vendorjs" %}
   <script type="text/javascript" src="{{ ASSET_URL }}"></script>
   {% endassets %}
+  {% assets "publicjs" %}
```

Comment:
I think this will make the publicjs bundle included in all templates, so I don't think you want to include it here.